mlflow/tracking/client.py

```diff
@@ -5928,3 +5929,139 @@ def delete_prompt(self, name: str) -> None:
 
         # For non-Unity Catalog registries, or if version check passes, delete the prompt
         return registry_client.delete_prompt(name)
+
+    # Webhook APIs
+    def create_webhook(
+        self,
+        name: str,
+        url: str,
+        events: list[Union[str, WebhookEvent]],
+        description: Optional[str] = None,
+        secret: Optional[str] = None,
+        status: Optional[Union[str, WebhookStatus]] = None,
+    ) -> Webhook:
+        """
+        Create a new webhook.
+
+        Args:
+            name: Unique name for the webhook.
+            url: Webhook endpoint URL.
+            events: List of event types that trigger this webhook. Can be strings or
+                WebhookEvent enums.
+                Valid event types:
+                - "REGISTERED_MODEL_CREATED"
+                - "MODEL_VERSION_CREATED"
+                - "MODEL_VERSION_TAG_SET"
+                - "MODEL_VERSION_TAG_DELETED"
+                - "MODEL_VERSION_ALIAS_CREATED"
+                - "MODEL_VERSION_ALIAS_DELETED"
+            description: Optional description of the webhook.
+            secret: Optional secret for HMAC signature verification.
+            status: Webhook status (defaults to ACTIVE). Can be string or WebhookStatus enum.
+                Valid statuses: "ACTIVE", "DISABLED"
+
+        Returns:
+            A :py:class:`mlflow.entities.webhook.Webhook` object representing the created webhook.
+        """
+        # Convert string events to WebhookEvent enums
+        converted_events = [
+            WebhookEvent(event) if isinstance(event, str) else event for event in events
+        ]
+        # Convert string status to WebhookStatus enum if needed
+        converted_status = None
+        if status is not None:
+            converted_status = WebhookStatus(status) if isinstance(status, str) else status
```

Comment:
Could we check status is instance of WebhookStatus if not string?

Reply:
We could but non-WebhookStatus `status` would cause `AttributeError: to_proto`.